src/side-navigation/side-navigation.scss

```diff
@@ -1,207 +1,137 @@
-    justify-content: center;
-    align-items: center;
-
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
-
-    > .iui-icon {
-      width: $iui-icons-small;
-      height: $iui-icons-small;
-
-      @include themed {
-        fill: t(iui-icons-color);
-      }
-    }
-
-    &:hover > .iui-icon {
-      @include themed {
-        fill: t(iui-icons-color-primary);
-      }
-    }
-
-    &:focus {
-      outline: none;
-    }
-
-    &:focus-visible {
-      @include themed {
-        box-shadow: inset 0 0 $iui-xs
-          rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-3));
-      }
-    }
-  }
-
-  > .iui-sidenav-content {
+  > .iui-content {
     display: flex;
-    flex-direction: row;
-    height: calc(100% - #{$iui-l});
+    flex-direction: column;
+    flex-grow: 1;
+    width: 100%;
 
-    > .iui-sidenav-items-container {
+    > .iui-top,
+    > .iui-bottom {
+      flex-grow: 1;
       display: flex;
       flex-direction: column;
+    }
 
-      > .iui-sidenav-items {
-        flex-grow: 1;
-        display: flex;
-        flex-direction: column;
-      }
+    > .iui-top > .iui-button:not(:last-of-type) {
+      border-bottom: none;
+    }
+    
+    > .iui-bottom {
+      justify-content: flex-end;
 
-      > .iui-sidenav-items > .iui-sidenav-item-container,
-      > .iui-bottom-container > .iui-sidenav-item-container {
-        margin-bottom: -1px;
-        @include iui-sidenav-item-container;
+      > .iui-button {
+        border-bottom: none;
       }
     }
   }
 
   &.iui-collapsed {
-    min-width: $iui-l * 2;
-    max-width: $iui-l * 2;
+    min-width: $iui-icons-large * 3;
+    max-width: $iui-icons-large * 3;
   }
 
   &.iui-expanded,
-  > .iui-sidenav-content > .iui-sidenav-items-container {
+  > .iui-content {
     max-width: $iui-3xl * 3;
-  }
-
-  &.iui-expanded > .iui-expand-container > .iui-icon {
-    transform: rotate(180deg);
-  }
-}
 
-@mixin iui-sidenav-item-container {
-  display: flex;
-  flex-direction: row;
-  user-select: none;
-  height: $iui-l * 2;
-  cursor: pointer;
-  align-items: center;
-  transition: box-shadow $iui-speed-fast ease;
-  text-decoration: none;
-  box-sizing: border-box;
-  border-right: none;
-  border-left: none;
-
-  @include themed {
-    border-top: 1px solid t(iui-color-background-5);
-    border-bottom: 1px solid t(iui-color-background-5);
-    background-color: t(iui-color-background-1);
-  }
-
-  &:not(:last-child) {
-    border-bottom: none;
-  }
-
-  > .iui-sidenav-item-icon {
-    padding: 0 $iui-icons-default;
-    flex-shrink: 0;
-    height: $iui-icons-default;
-    width: $iui-icons-default;
-    transition: fill $iui-speed-fast ease;
-
-    @include themed {
-      fill: t(iui-icons-color);
+    > .iui-expand > .iui-icon {
+      transform: scaleX(-1);
     }
   }
 
-  > .iui-sidenav-item-caption {
-    transition: color $iui-speed-fast ease;
-    overflow-wrap: break-word;
-    white-space: nowrap;
+  .iui-button {
```

Comment:
This could cause problems if we want to support submenus or custom content in the future.

I would add a class here like `.iui-sidenav-button`.

Reply:
@FlyersPh9 Please validate.

I've changed:
`.iui-button` --> `.iui-sidenav-button`
`.iui-content` --> `.iui-sidenav-content`
`.iui-expand` --> `.iui-sidenav-button.iui-expand`

This is another step towards #55